### ISSUE TITLE
Error with v0.12.1

### ISSUE THREAD

**Issue by SBRK** [paperjs/paper-jsdom#1]
I'm getting the following error after updating to 0.12.1:

```
TypeError: jsdom.JSDOM is not a constructor
    at Object.<anonymous> (C:\dev\service-optimizer-plan-generator\node_modules\paper-jsdom\node_modules\paper\dist\node\self.js:40:20)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (C:\dev\service-optimizer-plan-generator\node_modules\paper-jsdom\node_modules\paper\dist\paper-full.js:35:16)
    at Object.<anonymous> (C:\dev\service-optimizer-plan-generator\node_modules\paper-jsdom\node_modules\paper\dist\paper-full.js:17024:3)
```

Did you forget to bump the version of jsdom ?

**Comment by lehni** [comment]
Thank you for pointing this out. New version coming up shortly!

**Comment by lehni** [comment]
Released in v0.12.2
